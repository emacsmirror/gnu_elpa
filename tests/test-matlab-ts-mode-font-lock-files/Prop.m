% -*- matlab-ts -*-
classdef Prop
    properties
        p0
        p1 double;
        p2 double = 0;
        p3 (1,1) double;
        p4 (1,1) double = 0;
        p5 (1,1) double {mustBePositive} = 0;
    end
end
