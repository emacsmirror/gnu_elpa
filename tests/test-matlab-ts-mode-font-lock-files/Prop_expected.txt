c ccc ccccccccc ccc
kkkkkkkk ffff
    kkkkkkkkkk
        PP
        PP ttttttD
        PP tttttt o nD
        PP bnDnb ttttttD
        PP bnDnb tttttt o nD
        PP bnDnb tttttt bBBBBBBBBBBBBBBb o nD
    kkk
kkk
